p4-16/spec/P4-16-spec.mdk

```diff
@@ -1906,13 +1906,18 @@ Operations on values of type `match_kind` are described in Section
 ### The Boolean type { #sec-bool-type }
 
 The Boolean type `bool` contains just two values, `false` and `true`.
-Boolean values are not integers or bit-strings.
+Boolean values are not integers or bit-strings. Operations that can
+be performed on booleans are described in Section [#sec-bool-exprs].
 
 ### Strings { #sec-string-type }
 
-The type `string` represents strings.  There are no operations on
-string values; one cannot declare variables with a `string` type.
-Parameters with type `string` can be only directionless (see Section
+The type `string` represents strings. The values are either string
```

Comment:
It's a bit strange to think of concatenations as being values... Maybe "string-typed expressions are either string literals or concatenations of multiple string-typed expressions."?